.github/pull_request_template.md

```diff
@@ -2,14 +2,14 @@
 *Give a high-level description of the changes in this PR*
 
 ### Testing Done
-*Outline any testing that was done for these changes. This could be unit tests, integration tests, etc.*
+*Outline any testing that was done for these changes. This could be unit tests, integration tests, testing on the car, etc.*
 
 ### Resolved Issues
 *Link any issues that this PR resolved. Eg `resolves #1, #2, and #5` (note that they MUST be specified like this so Github can automatically close them then this PR merges)*
 
 ### Checklist
 *Please change `[]` to `[x]` when you are ready.*
-- [ ] I have read and followed the code standards detailed in http://svformulaep1.teams.apsc.ubc.ca:8090/display/TR/Workflow. I understand that not doing so will waste my reviewer's time.
+- [ ] I have read and followed the code standards detailed in http://svformulaep1.teams.apsc.ubc.ca:8090/display/TR/Workflow (*This will save time for both you and the reviewer!*).
```

Comment:
```suggestion
- [ ] I have read and followed the code standards detailed in http://svformulaep1.teams.apsc.ubc.ca:8090/display/TR/Workflow (*Otherwise I Am Small Smooth Brain Gang Small Smooth Brain Gang*).
```

Reply:
:thinking: